common/test/package.scala

```diff
@@ -73,6 +73,9 @@ trait ConfiguredTestSuite extends TestSuite with ConfiguredServer with Configure
 
 trait SingleServerSuite extends TestSuite with GuiceOneServerPerSuite with OneBrowserPerSuite with HtmlUnitFactory {
 
+  // Fixes `Failed to listen for HTTP on /0.0.0.0:19001`
+  System.setProperty("testserver.port", "0")
```

Comment:
what does this fix?

Reply:
it should fix these errors

```
sbt.ForkMain$ForkError: java.lang.RuntimeException: play.core.server.ServerListenException: Failed to listen for HTTP on /0.0.0.0:19001!
```

Reply:
Get that but why is anything trying to connect to `:19001`?

Reply:
It's the default listening port used by `GuiceOneServerPerSuite` when running the apps.

Reply:
see https://github.com/playframework/scalatestplus-play/blob/master/module/src/main/scala/org/scalatestplus/play/BaseOneServerPerSuite.scala